### DISCUSSION TITLE
"Server IP" field not clickable / focusable / editable?

### DISCUSSION THREAD

**Discussion by jonstelly** [input-leap/input-leap#1891] [Q&A] [answered]
The GUI doesn't let me tab or click into "server ip" field on the clients... is that intentional?  I know there's some auto-configure client stuff in synergy so is that maybe the default/only option now?

FWIW the auto-connect didn't work for me, I had to manually set the serverHostname in `~/.config/InputLeap/InputLeap.conf`, once I did that, restarted client, it connected and negotiated as expected.

This is on arch with input-leap-git AUR package and Fedora input-leap package, ipv4 and ipv6 network

**Comment by shymega** [comment, marked as answer]
We're refactoring the GUI currently, so I wouldn't rely on Git right now. There's a lot of PRs pending review.

**Reply by jonstelly** [reply]
Gotcha, thanks for the quick answer and your work on the project

**Reply by shymega** [reply]
@jonstelly Thanks!

**Comment by shymega** [comment]
@sithlord48 Do you think this is related to #1830 not being merged yet? Wondering if OP should open a bug report.

**Reply by sithlord48** [reply]
@shymega  No #1830 is completely stand alone from any changes we landed in that dialog.

I can not recreate this issue with the newest build.
There are two cases where you are unable to click in that field (by design) so be sure you these do not apply to you 
1. Input Leap is in Server mode. 
2. Input Leap is in Client mode with the "auto-config" checkbox enabled.
